scripts/installer

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Tekton Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# dashboard flavour
+OPENSHIFT="false"
+READONLY="false"
+
+# configuration default values
+DEBUG="false"
+INSTALL_NAMESPACE="tekton-pipelines"
+PIPELINES_NAMESPACE="tekton-pipelines"
+TRIGGERS_NAMESPACE="tekton-pipelines"
+LOGOUT_URL=""
+CSRF_SECURE_COOKIE="true"
```

Comment:
should default to `false` for dev

Reply:
ok, i will inverse the logic, false by default and an option to enable it ?

Reply:
fixed